Makefile

```diff
@@ -79,7 +80,7 @@ test: test-kdd test-etcd
 
 .PHONY: test-kdd
 ## Run template tests against KDD
-test-kdd: bin/confd bin/kubectl bin/bird bin/bird6 run-k8s-apiserver
+test-kdd: bin/confd bin/kubectl bin/bird bin/bird6 bin/allocate-ipip-addr bin/calicoctl run-k8s-apiserver
```

Comment:
why do we need allocate-ipip-addr for KDD backend?

Reply:
because we use the same templates irrespective of the backend.  the templates invoke the allocate-ipip-addr function if the IPIP pools change.